src/Photos/Enums.cs

```diff
@@ -114,7 +130,7 @@ public enum PHCollectionListSubtype : long {
 	[Mac (10,12, onlyOn64 : true)]
 	[Native]
 	public enum PHCollectionEditOperation : long {
-		None             = 0,
+		// None             = 0,
```

Comment:
You can't remove enum fields, it's a breaking change.
```suggestion
		None             = 0,
```